src/test/compile-fail/issue-11740.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Could you fix up the copyright dates to 2017? I don't know if it matters, but it seems wrong to back-date copyrights.